specification/schemas/ReturnMethodResource.json

```diff
@@ -0,0 +1,19 @@
+{
+  "type": "object",
+  "required": [
+    "type"
+  ],
+  "properties": {
+    "type": {
+      "type": "string",
+      "pattern": "^return-methods$",
+      "example": "return-methods"
+    },
+    "id": {
+      "type": "string",
+      "format": "uuid",
+      "pattern": "^[a-f\\d]{8}(-[a-f\\d]{4}){3}-[a-f\\d]{12}$",
+      "example": "7fac478b-b3c4-46e6-b29e-a31599d27a7a"
```

Comment:
Let's generate a different UUID for this new resource (now it's the same as a Return)